src/Faker/Provider/Payment.php

```diff
@@ -211,9 +211,10 @@ public function creditCardDetails($valid = true)
      * @param  integer $length      total length without country code and 2 check digits
      * @return string
      */
-    protected static function iban($countryCode, $prefix = '', $length = null)
+    public static function iban($countryCode, $prefix = '', $length = null)
     {
-        $countryCode = strtoupper($countryCode);
+        $countryCode = is_null($countryCode) ? array_rand(self::$ibanFormats) : strtoupper($countryCode);
```

Comment:
yep, `array_rand()` is verboten: it doesn't use the Mersenne Twister randomizer.